byte-buddy-dep/src/test/java/net/bytebuddy/description/type/TypeDescriptionForLoadedTypeOfTest.java

```diff
@@ -0,0 +1,151 @@
+package net.bytebuddy.description.type;
+
+import net.bytebuddy.dynamic.loading.ByteArrayClassLoader;
+import net.bytebuddy.dynamic.loading.ClassLoadingStrategy;
+import net.bytebuddy.matcher.ElementMatchers;
+import net.bytebuddy.test.utility.ClassFileExtraction;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class TypeDescriptionForLoadedTypeOfTest extends AbstractTypeDescriptionTest {
```

Comment:
A particular test is not necessary here, this is an implementation detail that should be able to be switched out without noticable changes.